index.html

```diff
@@ -7,30 +7,92 @@
       name="viewport"
       content="width=device-width, height=device-height, initial-scale=1.0"
     />
-  
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+   
     <!-- #### stylesheets ####  -->
     <link rel="stylesheet" href="css/main.css" />
     <title>Awesome Books</title>
   </head>
   <body>
-    
-    <main>
-      <h1 class="main-heading"> All Awesome Books</h1>
-      <div class="container">
+    <header>
+      <h3 class="header-logo"><i class="fas fa-book-open"></i> Awsome Books</h3>
+      <!-- #### navbar-section #### -->
+      <nav>
+        <ul class="desktop-menu">
+          <li><a class="desktop-item list a" href="#">List</a></li>
+          <li><a class="desktop-item new a" href="#">Add new</a></li>
+          <li><a class="desktop-item contact a" href="#">Contact</a></li>
+        </ul>
+       
+        <div class="mobile-nav">
+          <button class="btn-hamburger" type="button"><i class="fas fa-bars"></i></button>
+          <div class="mobile-menu-c">
+            <button class="btn-close" type="button"><i class="fas fa-times"></i></button>
+          <ul class="mobile-menu">
+          <li><a class="mobile-item list" href="#">List</a></li>
+          <li><a class="mobile-item new" href="#">Add new</a></li>
+          <li><a class="mobile-item contact" href="#">Contact</a></li>
+        </ul>
         </div>
+        </div>
+      </nav>
+     
+    </header>
 
-        <div class="form-container">
-          <hr>
-          
+    <main>
+      <div class="time"></div>
+     <!-- #### book-list-section #### -->
+      <div class="book-list-c">
+      <h1 class="main-heading">All Awesome Books</h1>
+      <div class="container"></div>
+      </div>
+
+      <div class="form-container">
+        <hr />
+
+      <!-- ####  add-book-section #### -->
         <h2>Add a new book</h2>
         <form class="form">
-          <input placeholder="Title" class="title" required><br><br>
-          <input placeholder="Author" class="author" required><br><br>
+          <input placeholder="Title" class="title" required /><br /><br />
+          <input placeholder="Author" class="author" required /><br /><br />
           <button type="submit" class="btn-add">Add</button>
         </form>
       </div>
-    
+
+      <!-- #### cobtact-section ##### -->
+      <section class="contact-section">
+        <h1 class="contact-heading">Contact Information</h1>
+        <p class="contact-details">
+          Do you have any quadtions, Or you just want to contact us<br />You can
```

Comment:
- [ ] Everything looks good just one issue I'm seeing is that when I'm opening the application for the first time or I'm refreshing the page, all sections are showing in one page. Please fix that and make sure only one section will open at a time.

Currently, your application looks like the below screenshot when I'm refreshing or opening it for the first time. Please fix that

![Screenshot from 2021-11-25 15-33-49](https://user-images.githubusercontent.com/22434660/143421748-e61355d4-d604-419d-a88d-8fb2176e0220.png)